src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -818,10 +832,12 @@ func (s *fileSystemSource) persistBootstrapIndexSegment(
 	// and we didn't bootstrap any more/less
 	requireFulfilled := expectedRanges.Copy()
 	requireFulfilled.Subtract(fulfilled)
-	exactStartEnd := min.Equal(blockStart) && max.Equal(blockStart.Add(blockSize))
+	exactStartEnd := max.Equal(blockStart.Add(blockSize))
```

Comment:
not part of your change but the comment above is missing a "we" -> "Check that we completely..."

Reply:
Done, TY.